app/src/main/res/values/strings.xml

```diff
@@ -115,17 +115,17 @@
     <string name="job">Job</string>
     <string name="phone_number">Phone number</string>
     <string name="street">Street</string>
-    <string name="zip_code">Zip code</string>
+    <string name="zip_code">ZIP code</string>
```

Comment:
As per https://en.wikipedia.org/wiki/ZIP_Code

Reply:
Oh ok, I didn't know. My country does not use ZIP codes, it uses postal codes